coral-incremental/src/main/java/com/linkedin/coral/incremental/TableStatistic.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2024 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.incremental;
+
+import java.util.Map;
+
+
+public class TableStatistic {
```

Comment:
Any reason this is not a nested class in `RelNodeCostEstimator`? Are we expecting to use this class outside of `RelNodeCostEstimator`?

Reply:
good catch, will move it inside the class.